.travis.yml

```diff
@@ -73,4 +73,5 @@ script:
   - git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
   - ./check_code_format.sh
   - catkin build local_planner global_planner --no-deps -v -i --catkin-make-args tests
-  - status=0 && for f in ~/catkin_ws/devel/lib/*/*-test; do valgrind --leak-check=full --track-origins=yes --error-exitcode=1 $f || status=1; done && exit $status
+  - status=0 && for f in ~/catkin_ws/devel/lib/*/*-test; do valgrind --leak-check=full --track-origins=yes --error-exitcode=1 $f || status=1; done
+  - status=0 && (roscore &) && for f in ~/catkin_ws/devel/lib/*/*-test-roscore; do valgrind --leak-check=full --track-origins=yes --error-exitcode=1 $f || status=1; done && exit $status
```

Comment:
This line we don't need the status=0, otherwise we overwrite any errors from the other test run